### DISCUSSION TITLE
Daily 12/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#26] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Problemas o Impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer instalé el nodejs y y el tsc. Hice el exámen nivelatorio de inglés y presenté el formulario correspondiente. Estuve revisando la documentación, pero no pude avanzar mucho.
Hoy instalaré Angular. Voy a continuar con la documentación. También quiero avanzar con los contenidos del ISPC. Además trataré de crear algunas historias de usuario.
Impedimentos: falta de conocimiento de algunos temas y falta de tiempo.

**Comment by Brian-Almada** [comment]
Ayer hice las tareas de ingles (El test y el formulario) y luego estuve revisando el github para ponerme al día. Hoy voy a estudiar un poco de Angular. También estoy pensando historias de usuario y nombre para el proyecto, aunque hasta ahora no se me ha ocurrido nada.

**Comment by AhumadaCeleste** [comment]
Aun no pude iniciar con Angular, ni con nada del contenido, dado que estuve trabajando con lo relacionado a BD del proyecto, creando isuue, historias de usuarios, e imágenes para login. Mañana espero poder iniciar con algo de la cursada

**Comment by rpgrca** [comment]
Estuve revisando el modelo de datos y haciendo correcciones, también estuve dejando comentarios en algunas de las user stories propuestas. También instalé e hice un par de pruebas en Angular con la guía de estudios, y miré por arriba la guía de OOP. Si el modelo de datos cierra podríamos empezar a armar los diagramas correspondientes, como me quedé sin hacer el de la base de datos podría tomar el diagrama de clases.

**Reply by NataliaAlvarezIspc** [reply]
Si Roberto, hace el diagrama de clases.

**Comment by BraianTroncoso** [comment]
Ayer, refactoricé el código viejo del antiguo proyecto y lo hice mas intuitivo y respetando psicología de color. A demás tambien agregué una categoría nueva, todo solamente como una sugerencia pero que me sirve de practica.
Hoy voy seguir practicando conceptos sobre POO y leer mas documentación. A demás de codificar un poco.
El único impedimento, las ganas.

**Comment by Tarias89** [comment]
Ayer hice el test de ingles y subí el certificado al formulario.
Hoy sigo con el material de estudio y con la instalación de node y typescript. Angular me da error.
Mis impedimentos son falta de practica, de tiempo y desconocimiento de angular.

**Comment by KAAcevedo** [comment]
Hice el EF SET Certificate de 50 min y presenté el formulario. Estuve revisando un poco todo.
También quiero avanzar con los contenidos del ISPC, vengo un poco atrasada. Me gustaría mañana revisar nuevamente el repo y crear algunas _User stories_.
